### DISCUSSION TITLE
the function of triton_start_loc

### DISCUSSION THREAD

**Discussion by zhaoofengyang** [sgl-project/sglang#1334] [Q&A]
when the sglang do decode, for examples， there are 2 seqs，the triton_start_loc will be [0, len(1st seq)]
could someone can explain the function, how to use it?

**Comment by merrymercy** [comment]
It is used by triton attention kernels https://github.com/sgl-project/sglang/blob/2561ed012ce10e109ac888f7e9e7ffe44ccb4a94/python/sglang/srt/layers/decode_attention.py
https://github.com/sgl-project/sglang/blob/2561ed012ce10e109ac888f7e9e7ffe44ccb4a94/python/sglang/srt/layers/extend_attention.py
